src/basics.php

```diff
@@ -51,7 +51,13 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1
+                && $trace[1]['function'] === 'dd'
```

Comment:
@markstory It is debug()+die()
See https://github.com/cakephp/cakephp/blob/master/src/basics.php#L173